analysis/neutron/openmc_model.py

```diff
@@ -330,38 +474,214 @@ def baby_model():
         air,
         epoxy,
         he,
+        HDPE,
+        diamond,
+        Zr,
+        Nb,
     ]
 
     # BABY coordinates
     x_c = 587  # cm
     y_c = 60  # cm
     z_c = 100  # cm
-    sphere, cllif_cell, cells = baby_geometry(x_c, y_c, z_c)
+    (
+        experimental_lab,
+        cllif_cell,
+        diamond_detect_cell,
+        act_foils_zr_cell,
+        act_foils_nb_cell,
+        cells,
+    ) = baby_geometry(x_c, y_c, z_c)
+
+    # The coordinates of the source in the Nuclear Vault used in a separate experiment
+    x_c_ns = 500.5
+    y_c_ns = 225.0
+    z_c_ns = 138.0
 
     ############################################################################
     # Define Settings
 
     settings = openmc.Settings()
 
     src = A325_generator_diamond((x_c, y_c, z_c - 5.635), (1, 0, 0))
+    # The underlying source is part of a separate experiment carried out inside the Nuclear Vault
+    # src = A325_generator_diamond((x_c_ns - 20.5, y_c_ns, z_c_ns), (1, 0, 0))
     settings.source = src
     settings.batches = 100
     settings.inactive = 0
     settings.run_mode = "fixed source"
-    settings.particles = int(1e4)
+    settings.particles = int(1e5)
     settings.output = {"tallies": False}
-    settings.photon_transport = False
+    settings.photon_transport = True
```

Comment:
Don't we need photon cross sections in that case? is this covered by `openmc_data_downloader` or `irdff`? 

pinging @shimwell